.github/workflows/ci.yml

```diff
@@ -112,4 +112,3 @@ jobs:
       python-repo-path: ${{github.event.pull_request.head.repo.full_name}}
       version: ${{github.event.pull_request.head.ref}}
       version-is-repo-ref: true
-      features-repo-ref: python-version-upgrade
```

Comment:
This was stale: there is nothing in `features.git@python-version-upgrade` that is not in `features.git@main`.